### PR TITLE
verticalRange as a Number not Double

### PR DESCRIPTION
**Pull request by pmbohm**
Required for https://github.com/aodn/geoserver-build/pull/274

### CODE REVIEW

File: aggregation-worker/src/main/java/au/org/emii/aggregationworker/AggregationRunner.java [hunk 1]
```diff
@@ -35,19 +37,12 @@
...
 
 import java.io.IOException;
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.nio.file.Paths;
-import java.util.Collection;
-import java.util.HashMap;
-import java.util.LinkedHashSet;
-import java.util.List;
-import java.util.Map;
```
Comment (jonescc) [inline aggregation-worker/src/main/java/au/org/emii/aggregationworker/AggregationRunner.java]: Getting compilation errors for these being missing:

```
[ERROR] /home/craigj/git-repos/aodn/aws-wps/aggregation-worker/src/main/java/au/org/emii/aggregationworker/AggregationRunner.java:[200,13] cannot find symbol
  symbol:   class List
  location: class au.org.emii.aggregationworker.AggregationRunner
[ERROR] /home/craigj/git-repos/aodn/aws-wps/aggregation-worker/src/main/java/au/org/emii/aggregationworker/AggregationRunner.java:[224,71] cannot find symbol
  symbol:   class LinkedHashSet
  location: class au.org.emii.aggregationworker.AggregationRunner
[ERROR] /home/craigj/git-repos/aodn/aws-wps/aggregation-worker/src/main/java/au/org/emii/aggregationworker/AggregationRunner.java:[231,17] cannot find symbol
  symbol:   class HashMap
  location: class au.org.emii.aggregationworker.AggregationRunner
[ERROR] /home/craigj/git-repos/aodn/aws-wps/aggregation-worker/src/main/java/au/org/emii/aggregationworker/AggregationRunner.java:[231,57] cannot find symbol
  symbol:   class HashMap
  location: class au.org.emii.aggregationworker.AggregationRunner
[ERROR] /home/craigj/git-repos/aodn/aws-wps/aggregation-worker/src/main/java/au/org/emii/aggregationworker/AggregationRunner.java:[267,21] cannot find symbol
  symbol:   class Map
  location: class au.org.emii.aggregationworker.AggregationRunner
[ERROR] /home/craigj/git-repos/aodn/aws-wps/aggregation-worker/src/main/java/au/org/emii/aggregationworker/AggregationRunner.java:[267,54] cannot find symbol
  symbol:   class HashMap
  location: class au.org.emii.aggregationworker.AggregationRunner
[ERROR] /home/craigj/git-repos/aodn/aws-wps/aggregation-worker/src/main/java/au/org/emii/aggregationworker/AggregationRunner.java:[366,13] cannot find symbol
  symbol:   class Map
  location: class au.org.emii.aggregationworker.AggregationRunner
[ERROR] /home/craigj/git-repos/aodn/aws-wps/aggregation-worker/src/main/java/au/org/emii/aggregationworker/AggregationRunner.java:[367,13] cannot find symbol
  symbol:   class Collection
  location: class au.org.emii.aggregationworker.AggregationRunner
```

**Comment by pmbohm** [comment]
@jonescc Replaced the imports

??

**Comment by pmbohm** [comment]
@jonescc Peter would like this to be demoed. I guess this is possible if you are bringing this up in AWS to test?

**Comment by jonescc** [comment]
Getting exception: Exception occurred during aggregation : For input string: "${Z_MAX}" running integration tests.   I'll have a look

**Comment by jonescc** [comment]
So that was a deployment issue.  Works fine after I sorted that out.

**Pull request merged**